Lil-Peep/src/styles/modules/featured-kits.scss

```diff
@@ -0,0 +1,448 @@
+}
+
+@media only screen and (width: 768px) {
+  .big-banner {
+    background-position: center;
+    background-size: cover;
+  }
+
+  .big-banner-content p {
+    margin-bottom: 20px;
+  }
+
+  .small-banner {
+    background-position: bottom;
+    background-size: cover;
+  }
+
+  .mid-banner-saylor {
+    background-position: center;
+    background-size: cover;
+  }
+
+  .mid-banner-archeologist {
+    background-position: top;
+  }
+}
+
+@media only screen and (min-width: 769px) and (max-width: 1279px) {
+  .flex-grid {
+    display: flex;
+    margin-bottom: 20px;
+  }
+
+  .big-banner {
+    background-position: left top;
+    background-size: cover;
+    display: flex;
+    flex: 2;
+    height: 400px;
+    justify-content: flex-end;
+    margin-right: 20px;
+  }
+
+  .big-banner-content {
+    align-items: center;
+    background: linear-gradient(270deg,
+      rgb(251,239,227),
+      rgba(251,239,227, 1) 80%,
+      rgba(251,239,227, 0));
+    display: flex;
+    flex-direction: column;
+    height: 400px;
+    justify-content: center;
+    padding-right: 10px;
+    width: 50%;
+  }
+
+  .big-banner-content .view-in-detail {
+    width: 200px;
+  }
+
+  .big-banner-content p {
+    font-size: 15px;
+    margin-bottom: 30px;
+    margin-top: 20px;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+  .big-banner-content h1 {
+    font-size: 25px;
+  }
+
+  .big-banner-content button {
+    padding-left: 30px;
+    padding-right: 30px;
+  }
+
+  .small-banner {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    justify-content: center;
+  }
+
+  .small-banner p {
+    font-size: 30px;
+    margin: 0;
+  }
+
+  .big-text p {
+    font-size: 60px;
+    font-weight: bold;
+    margin: 0;
+  }
+
+  .mid-banner {
+    flex: 1;
+    height: 200px;
+    justify-content: flex-end;
+  }
+
+  .mid-banner-content {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 200px;
+    justify-content: center;
+    padding-left: 10px;
+    padding-right: 10px;
+    width: 50%;
+  }
+
+  .mid-banner-content button {
+    width: 100%;
+  }
+
+  .mid-banner-content p {
+    font-size: 13px;
+  }
+
+  .mid-banner-saylor {
+    background-position: right;
+    background-repeat: no-repeat;
+    background-size: 400px 300px;
+  }
+
+  .mid-banner-saylor .mid-banner-content {
+    background: linear-gradient(90deg,
+      rgb(239, 239, 239),
+      rgb(239, 239, 239) 50%,
+      rgb(239, 239, 239) 90%,
+      rgba(239, 239, 239, 0));
+  }
+
+  .mid-banner-archeologist {
+    background-position: right;
+    background-repeat: no-repeat;
+    background-size: 400px 300px;
+    margin-left: 20px;
+  }
+
+  .mid-banner-archeologist .mid-banner-content {
+    background: linear-gradient(90deg,
+      rgb(189, 152, 107),
+      rgb(189, 152, 107) 50%,
+      rgb(189, 152, 107) 90%,
+      rgba(189, 152, 107, 0));
+  }
+}
+
+@media only screen and (min-width: 1280px) {
```

Comment:
Debes implementar los breakpoints de esta forma:

$break-small: 320px;
$break-large: 1200px;

.profile-pic {
  float: left;
  width: 250px;
  @media screen and (max-width: $break-small) {
    width: 100px;
    float: none;
  }
  @media screen and (min-width: $break-large) {
    float: right;
  }
}